recipes/graphene-django/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "graphene-django" %}
+{% set version = "2.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6e64bc9df1e4d4be247aedc63f61047cae673156198e433110bf87ae2d072a83
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - django >=1.11
+    - graphene >=2.1.7,<3
+    - graphql-core >=2.1.0,<3
+    - pip
+    - promise >=2.1
+    - python
+    - singledispatch >=3.4.0.3
+    - six >=1.10.0
+  run:
+    - django >=1.11
+    - graphene >=2.1.7,<3
+    - graphql-core >=2.1.0,<3
+    - promise >=2.1
+    - python
+    - singledispatch >=3.4.0.3
+    - six >=1.10.0
```

Comment:
These pins are fine, but you should double-check the upstream package to make sure they are correct according to the docs. PyPi metadata can be unreliable.

Reply:
Yeah, those are the required packages to be installed and pinned according to upstream [setup.py](https://github.com/graphql-python/graphene-django/blob/master/setup.py) file.